espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/ScreenshotHandler.java

```diff
@@ -16,15 +16,27 @@
 
 package io.appium.espressoserver.lib.handlers;
 
+import android.graphics.Bitmap;
+import android.util.Base64;
+
+import java.io.ByteArrayOutputStream;
+
+import androidx.test.runner.screenshot.ScreenCapture;
+import androidx.test.runner.screenshot.Screenshot;
 import io.appium.espressoserver.lib.handlers.exceptions.AppiumException;
 import io.appium.espressoserver.lib.helpers.ScreenshotsHelper;
 import io.appium.espressoserver.lib.model.AppiumParams;
 import io.appium.espressoserver.lib.viewaction.ViewGetter;
 
-public class Screenshot implements RequestHandler<AppiumParams, String> {
+public class ScreenshotHandler implements RequestHandler<AppiumParams, String> {
 
     @Override
     public String handle(AppiumParams params) throws AppiumException {
-        return new ScreenshotsHelper(new ViewGetter().getRootView()).getScreenshot();
```

Comment:
Please also refactor screenshot helper and include this feature to ElementScreenshot handler